conans/test/conftest.py

```diff
@@ -163,6 +163,8 @@
 
 
 # TODO: Make this match the default tools (compilers) above automatically
+arch = platform.machine()
+arch_setting = "armv8" if arch in ["arm64", "aarch64"] else arch
```

Comment:
On Windows/ARM, I believe the _native_ Python interpreter reports `ARM64` (all capitals). Maybe for a future update, since this is not currently commonplace and we have no way of testing it.

Also apparently a native arm64 Python interpreter is a very recent thing :D